zcash/Cargo.toml

```diff
@@ -33,6 +34,8 @@ secp256k1 = { version = "0.15.0" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
 sha2 = { version = "0.7" }
+rust-crypto = { version = "0.2" }
```

Comment:
Why do we need to import two sha2 implementations in one crate? Can we pick one?